node/test/utils/toFieldArrayFormat.unit.test.ts

```diff
@@ -0,0 +1,22 @@
+import {expect} from "chai";
```

Comment:
Is there any particular reason to use `mocha + chai` with Node and `vtex/test-tools(jest)` on with React? Why not use the same on both?

Reply:
There is no particular reason for this difference. I just used `mocha + chai` because I know how to configure some useful things I added to the repo - .env i.e.